### ISSUE TITLE
contopac/atmos.sun_marcs_t5777_4.44_0.00_vmic1_newopac

### ISSUE THREAD

**Issue by sagiph-weizmann** [bertrandplez/Turbospectrum_NLTE#14]
Trying to verify turbospectrum_NLTE is running on my laptop before moving to TSFitPy.
Running script-TS-v20.0-babsma to the Turbospectrum_NLTE directory, and getting the following missing file error:

At line 331 of file ../source/babsma.f (unit = 13)
Fortran runtime error: Cannot open file 'contopac/atmos.sun_marcs_t5777_4.44_0.00_vmic1_newopac': No such file or directory

Please advise.
Thanks for the help!
S.

**Comment by stormnick** [comment]
Hi, have you tried creating `contopac` folder yourself beforehand? I think the script doesn't create the folder if it doesn't exist and thus crashes

**Comment by sagiph-weizmann** [comment]
Thanks for the fast reply!
That did move me one step further.
I than get the following error:
At line 506 of file ../source/babsma.f (unit = 12)
Fortran runtime error: Cannot open file '/Users/benamis/tools/Turbospectrum_NLTE/COM/TEST-data/atmos.sun_marcs_t5777_4.44_0.00_vmic1_new': No such file or directory

ISo I created also the library Turbospectrum_NLTE/COM/TEST-data and have copied there the file atmos.sun_marcs_t5777_4.44_0.00_vmic1_newopac from the contopac librar, changing the file name to atmos.sun_marcs_t5777_4.44_0.00_vmic1_new

I than get the following error:
At line 507 of file ../source/babsma.f (unit = 12, file = '/Users/benamis/tools/Turbospectrum_NLTE/COM/TEST-data/atmos.sun_marcs_t5777_4.44_0.00_vmic1_new')
Fortran runtime error: End of file



I am sure it is extremely trivial - and I appreciate the time you take to help newbees !

**Comment by stormnick** [comment]
I don't think you are supposed to copy the file `atmos.sun_marcs_t5777_4.44_0.00_vmic1_newopac` and rename it, the code seems to expect the actual atmosphere file `atmos.sun_marcs_t5777_4.44_0.00_vmic1_new`. Do you have that atmosphere? It seems like the script expects you to use it by default, but you probably don't have it (it doesn't seem to be included by default here). You can download it from the MARCS website (https://marcs.astro.uu.se), but I can also directly give you the file for now (just unzip it):
[p5777_g+4.4_m0.0_t01_st_z+0.00_a+0.00_c+0.00_n+0.00_o+0.00_r+0.00_s+0.00.mod.zip](https://github.com/bertrandplez/Turbospectrum_NLTE/files/13904591/p5777_g%2B4.4_m0.0_t01_st_z%2B0.00_a%2B0.00_c%2B0.00_n%2B0.00_o%2B0.00_r%2B0.00_s%2B0.00.mod.zip)

**Comment by sagiph-weizmann** [comment]
Seems like another step forward. It is reading the file, but now I get the following:

 56 -15.45  -7.92  -9.06 -16.86  -8.76  -9.51  -7. 
 trying multi format
                                                   
At line 552 of file ../source/babsma.f (unit = 12, file = '/Users/benamis/tools/Turbospectrum_NLTE/COM/TEST-data/p5777_g+4.4_m0.0_t01_st_z+0.00_a+0.00_c+0.00_n+0.00_o+0.00_r+0.00_s+0.00.mod')
Fortran runtime error: End of file

**Comment by bertrandplez** [comment]
Hi,

If you use this file, this is a MARCS formatted file and you should have 

###########
# model atmosphere. Various formats allowed. Only MARCS can be binary or ascii. Others are ascii
#
'MODELINPUT:' '$mpath/${MODEL}'
'MARCS-FILE:' '.true.'
#

Sorry this is not yet fully automatised and MARCS files do have a special status.
Cheers,
Bertrand

**Comment by sagiph-weizmann** [comment]
Thanks Bertrand.
It did solve the babsma part, but I get a SIGSEGV segmentation fault in the bsyn part.

I am using gf compiler, and an i9 processor on a MAC.
Is there any information I can share that might be needed to check what I am doing wrong ?

Thanks again.
S.

**Comment by bertrandplez** [comment]
I need more information to understand the problem.
Can you send the log file directly to my email address ***@***.*** ***@***.***>
along with the script you use?
Bertrand

**Comment by sagiph-weizmann** [comment]
Hi Bertrand.
github scrambled the email address.
Please use the following link to email me, and I will mail the script and output file to this address.

Thanks again!
Sagi.

https://www.weizmann.ac.il/physics/dr-sagi-ben-ami
